src/m365/teams/commands/app/app-install.ts

```diff
@@ -133,22 +132,22 @@ class TeamsAppInstallCommand extends GraphCommand {
       debug: args.options.debug,
       verbose: args.options.verbose
     };
+    try {
```

Comment:
We already have a try catch in the commandAction method. Do we really need an extra one?

Reply:
oh wait, I see now :-) I missed the context...

Reply:
Totally missed your comment. Sorry